include/qpid/dispatch/atomic.h

```diff
@@ -205,6 +205,9 @@ static inline void sys_atomic_destroy(sys_atomic_t *ref)
 
 #endif
 
+#define    SET_ATOMIC_FLAG(flag)  sys_atomic_set(flag, 1)
```

Comment:
Why not a CLEAR_ATOMIC_FLAG?

Reply:
Introduced a CLEAR_ATOMIC_FLAG macro and the http adaptor now uses it to clear the flags once a new connection to the server is established.